test/Microsoft.AspNet.Authentication.Test/OpenIdConnect/Extensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Authentication.OpenIdConnect;
+using Microsoft.AspNet.Builder;
+using Microsoft.Framework.Logging;
+using Microsoft.Framework.OptionsModel;
+using Microsoft.Framework.WebEncoders;
+
+namespace Microsoft.AspNet.Authentication.Tests.OpenIdConnect
+{
+    /// <summary>
+    /// Extension specifies <see cref="CustomOpenIdConnectAuthenticationMiddleware"/> as the middleware.
+    /// </summary>
+    public static class OpenIdConnectAuthenticationExtensions
```

Comment:
The purpose of these tests is at the Unit / Functional level. Once I am satisfied that this level is functioning correctly, I will bump up to the next level.